.github/workflows/build-and-test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: cargo build --release
 
       - name: Test Cargo crates
-        run: cargo test
+        run: cargo test --all-features
```

Comment:
I had originally added an optional `assets` feature to the `ic-http-certification` crate and then needed this flag to run tests that were behind the feature flag. That's not needed anymore but I think it's good to keep this anyway.